src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -764,6 +764,30 @@ def test_functionapp_consumption_e2e(self, resource_group, storage_account):
 
         self.cmd('functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
 
+class FunctionAppOnLinux(ResourceGroupVCRTestBase):
```

Comment:
Please do not use `ResourceGroupVCRTestBase` any more.

Reply:
I just copied another test. Is there a doc somewhere that talks about writing tests or a sample test I can look at?